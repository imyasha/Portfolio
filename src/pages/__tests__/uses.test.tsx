import React from 'react';
import * as Gatsby from 'gatsby';
import 'jest-styled-components';
import { render } from '@testing-library/react';
import Uses from '../uses';

const useStaticQuery = jest.spyOn(Gatsby, 'useStaticQuery');
useStaticQuery.mockImplementation(() => ({
  site: {
    siteMetadata: {
      description: '',
      title: 'Uses',
      lang: 'en',
      meta: [],
      keywords: [],
    },
  },
}));

test('renders correctly', () => {
  const { container } = render(<Uses />);
  expect(container.firstChild).toMatchSnapshot();
});
