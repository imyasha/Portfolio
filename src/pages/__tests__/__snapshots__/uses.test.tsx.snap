// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  width: 1140px;
  max-width: 94%;
  margin: 0 auto;
}

.c1 {
  font-size: 3rem;
  margin: 0;
}

.c2 {
  padding: 0 1rem;
  margin: 0;
  list-style: square;
}

.c2:last-of-type {
  margin-bottom: 3rem;
}

@media print {
  .c1 {
    font-size: 22pt;
  }
}

<div
  class="c0"
>
  <h1
    class="c1"
  >
    Jacob uses
  </h1>
  <p>
    Welcome to my "Uses" page, that lists some of the hardware and software I use on a daily basis.
  </p>
  <h2>
    Hardware
  </h2>
  <ul
    class="c2"
  >
    <li>
      <strong>
        <a
          href="https://www.amazon.co.uk/gp/product/B0821M3WQ5?ie=UTF8&tag=jakeherp-21&camp=1634&linkCode=xm2&creativeASIN=B0821M3WQ5"
          rel="noopener noreferrer"
          target="_blank"
        >
          Macbook Pro 16"
        </a>
      </strong>
       
      (2.4 GHz i9 CPU, 32 GB Ram, 2TB storage)
    </li>
    <li>
      <strong>
        <a
          href="https://www.amazon.co.uk/gp/product/B07BR9ZBZ4?ie=UTF8&tag=jakeherp-21&camp=1634&linkCode=xm2&creativeASIN=B07BR9ZBZ4"
          rel="noopener noreferrer"
          target="_blank"
        >
          Apple Magic Keyboard 2
        </a>
      </strong>
    </li>
    <li>
      <strong>
        <a
          href="https://www.amazon.co.uk/gp/product/B07NJ2BCC2?ie=UTF8&tag=jakeherp-21&camp=1634&linkCode=xm2&creativeASIN=B07NJ2BCC2"
          rel="noopener noreferrer"
          target="_blank"
        >
          Apple Magic Trackpad 2
        </a>
      </strong>
    </li>
    <li>
      <strong>
        <a
          href="https://www.amazon.co.uk/gp/product/B0845MWTS1?ie=UTF8&tag=jakeherp-21&camp=1634&linkCode=xm2&creativeASIN=B0845MWTS1"
          rel="noopener noreferrer"
          target="_blank"
        >
          LG 34WN80C-B 34" ultra-wide monitor, connected via USB-C
        </a>
      </strong>
       
      4K monitor
    </li>
    <li>
      <strong>
        <a
          href="https://www.amazon.co.uk/gp/product/B01F01DRW6?ie=UTF8&tag=jakeherp-21&camp=1634&linkCode=xm2&creativeASIN=B01F01DRW6"
          rel="noopener noreferrer"
          target="_blank"
        >
          Rain Design mStand
        </a>
      </strong>
    </li>
    <li>
      <strong>
        <a
          href="https://www.amazon.co.uk/gp/product/B07XL8WJK8?ie=UTF8&tag=jakeherp-21&camp=1634&linkCode=xm2&creativeASIN=B07XL8WJK8"
          rel="noopener noreferrer"
          target="_blank"
        >
          iPhone 11 Pro
        </a>
      </strong>
       
      (256 GB)
    </li>
    <li>
      <strong>
        <a
          href="https://www.amazon.co.uk/gp/product/B07Q9MJKBV?ie=UTF8&tag=jakeherp-21&camp=1634&linkCode=xm2&creativeASIN=B07Q9MJKBV"
          rel="noopener noreferrer"
          target="_blank"
        >
          Bose 700
        </a>
      </strong>
       
      headphones
    </li>
    <li>
      <strong>
        <a
          href="https://www.amazon.co.uk/gp/product/B07ZPML7NP?ie=UTF8&tag=jakeherp-21&camp=1634&linkCode=xm2&creativeASIN=B07ZPML7NP"
          rel="noopener noreferrer"
          target="_blank"
        >
          Apple AirPods Pro
        </a>
      </strong>
       
      for on the go
    </li>
    <li>
      <strong>
        <a
          href="https://www.amazon.co.uk/gp/product/B07YSXZ7QD?ie=UTF8&tag=jakeherp-21&camp=1634&linkCode=xm2&creativeASIN=B07YSXZ7QD"
          rel="noopener noreferrer"
          target="_blank"
        >
          Nest Wifi
        </a>
      </strong>
       
      Mesh Network
    </li>
    <li>
      <strong>
        <a
          href="https://store.google.com/product/google_home"
          rel="noopener noreferrer"
          target="_blank"
        >
          Google Home
        </a>
      </strong>
       
      to manage my smart home
    </li>
  </ul>
  <h2>
    Development Tools
  </h2>
  <ul
    class="c2"
  >
    <li>
      <strong>
        <a
          href="https://code.visualstudio.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          VS Code
        </a>
      </strong>
       
      with the
       
      <strong>
        <a
          href="https://marketplace.visualstudio.com/items?itemName=sdras.night-owl"
          rel="noopener noreferrer"
          target="_blank"
        >
          Night Owl
        </a>
      </strong>
       
      theme and
       
      <strong>
        <a
          href="https://www.typography.com/fonts/operator/styles/operatormono"
          rel="noopener noreferrer"
          target="_blank"
        >
          Operator Mono
        </a>
      </strong>
       
      font
    </li>
    <li>
      <strong>
        <a
          href="https://www.iterm2.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          iTerm 2
        </a>
      </strong>
       
      Terminal
    </li>
    <li>
      <strong>
        <a
          href="https://www.sourcetreeapp.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Sourcetree
        </a>
      </strong>
       
      for version control beyond the command line
    </li>
    <li>
      <strong>
        <a
          href="https://insomnia.rest/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Insomnia
        </a>
      </strong>
       
      to test REST and GraphQL API's
    </li>
  </ul>
  <h2>
    Design Tools
  </h2>
  <ul
    class="c2"
  >
    <li>
      <strong>
        <a
          href="https://www.sketch.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Sketch
        </a>
      </strong>
    </li>
    <li>
      <strong>
        <a
          href="https://www.figma.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Figma
        </a>
      </strong>
    </li>
    <li>
      <strong>
        <a
          href="https://affinity.serif.com/en-gb/photo/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Affinity Photo
        </a>
      </strong>
      ,
       
      <strong>
        <a
          href="https://affinity.serif.com/en-gb/designer/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Affinity Designer
        </a>
      </strong>
      ,
       
      <strong>
        <a
          href="https://affinity.serif.com/en-gb/publisher/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Affinity Publisher
        </a>
      </strong>
    </li>
  </ul>
  <h2>
    Productivity
  </h2>
  <ul
    class="c2"
  >
    <li>
      <strong>
        <a
          href="https://www.notion.so/?r=008f1f1395694a2594659c43f8d666d8"
          rel="noopener noreferrer"
          target="_blank"
        >
          Notion
        </a>
      </strong>
       
      for notes and to-do lists
    </li>
    <li>
      <strong>
        <a
          href="https://www.atlassian.com/software/jira"
          rel="noopener noreferrer"
          target="_blank"
        >
          Jira
        </a>
      </strong>
       
      for project management
    </li>
    <li>
      <strong>
        <a
          href="https://slack.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Slack
        </a>
      </strong>
       
      for communication
    </li>
    <li>
      <strong>
        <a
          href="https://gsuite.google.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          G-Suite
        </a>
      </strong>
       
      for email, calendars and Google Drive
    </li>
  </ul>
  <h2>
    Various other software
  </h2>
  <ul
    class="c2"
  >
    <li>
      <strong>
        <a
          href="https://www.google.com/chrome/canary/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Google Chrome
        </a>
      </strong>
       
      (Canary) for development
    </li>
    <li>
      <strong>
        <a
          href="https://1password.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          1Password
        </a>
      </strong>
       
      as my password manager
    </li>
    <li>
      <strong>
        <a
          href="https://www.backblaze.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Backblaze
        </a>
      </strong>
       
      for constant backups of my machines
    </li>
    <li>
      <strong>
        <a
          href="https://magnet.crowdcafe.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Magnet
        </a>
      </strong>
       
      to manage windows
    </li>
  </ul>
</div>
`;
